src/Purebred/Config.hs

```diff
@@ -234,3 +236,36 @@ defaultConfig = do
         [ usePlugin Purebred.Plugin.UserAgent.plugin
         ]
     }
+
+-- | Replace some special tags with ASCII chars.
+--
+-- * @flagged@ → @!@
+-- * @attachment@ → @A@
+-- * @replied@ → @r@
+--
+tagReplacementMapAscii :: M.Map T.Text T.Text
+tagReplacementMapAscii = M.fromList
+  [ ("flagged", "!")
+  , ("attachment", "A")
+  , ("replied", "r")
+  ]
+
+-- | Replace some special tags with emoji.
+--
+-- * @flagged@ → 📌
+-- * @attachment@ → 📎
+-- * @replied@ → ↩️
+--
+tagReplacementMapEmoji :: M.Map T.Text T.Text
+tagReplacementMapEmoji = M.fromList
+  [ ("flagged", "📌")
+  , ("attachment", "📎")
```

Comment:
can we add `inbox` and `archive` too? :D

Reply:
`archive` doesn't seem to have any special meaning to notmuch: https://notmuchmail.org/special-tags/.
So I'm inclined to omit it.  You can add it via custom config.

`inbox` sure, what emoji and ascii char do you want?

Reply:
What do you think about: :inbox_tray: and `I` simply as an ASCII representation. Suppose we would want to keep the ASCII representation somewhat memorable.

Reply:
OK, PR updated, give it a spin.

An alternative @inbox@ emoji would be https://emojipedia.org/incoming-envelope/ but I'll let you decide.